### ISSUE TITLE
Is boundary able to check cross-boundary calls via render function.

### ISSUE THREAD

**Issue by Skoda091** [sasa1977/boundary#30]
### Situation

Concerning that, I have defined two boundaries
```elixir
defmodule MySystem.Account do
 use Boundary, top_level?: true, deps: [], exports: []
end
```
```elixir
defmodule MySystem.Blog do
 use Boundary, top_level?: true, deps: [], exports: []
end
```
I do a cross-boundary call via [render_many/4](https://hexdocs.pm/phoenix/Phoenix.View.html#render_many/4)
```elixir
defmodule MySystem.Account.UserView do
 ...
 def render("index.json", users) do
    %{users: render_many(users,  MySystem.Blog.UserView, "user.json"}
  end
 ...
end
```
Is boundary able to generate `forbidden call to MySystem.Blog.UserView` warnings in such a case?

**Comment by sasa1977** [comment]
Currently no. Boundary will only warn about explicit function call (`Foo.bar()`). Referencing a module without calling it (e.g. `apply(Foo, :bar, [])` isn't reported.

I think that this is technically possible, but it needs to be tried out.

**Comment by Skoda091** [comment]
That would be great if this would be possible. I will try it out in my spare time.

**Comment by sasa1977** [comment]
I've just published version 0.8.0 where this is possible. However, it's not turned on by default. See [here](https://hexdocs.pm/boundary/Boundary.html#module-alias-references) for details.
